### ISSUE TITLE
Markdown Alt text on images not recognized on some cases

### ISSUE THREAD

**Issue by hellpe** [writeas/writeas-go#24]
(not sure if this is the right place to post this, I've copied my post from https://discuss.write.as/t/markdown-alt-text-on-images-not-recognized-on-some-cases/2818)

I’ve noticed that on some of the images on [my blog](https://write.as/ailepet/), the alt text (i.e. the part between `![` and `](https://my-image-url.png)` is missing from the generated HTML code.

To be more precise, consider the following Markdown:
`![Capture d'écran de Our House de GameGrumps (url)](https://système.pointecouteau.com/writeas/superliminal.png)`

When I save my post with this code, it is correctly converted:
`<img src="https://syst%C3%A8me.pointecouteau.com/writeas/superliminal.png" alt="Capture d'écran de Our House de GameGrumps (url)">`

Howewer, for those two Markdown sentences, the generated HTML omits the “alt” tag entirely for some reason:

- `![Capture d'écran de "Our House" de GameGrumps](https://système.pointecouteau.com/writeas/superliminal.png)`
- `![Capture d'écran de Our House de GameGrumps (https://www.youtube.com/watch?v=k55FYtqtXXU)](https://système.pointecouteau.com/writeas/superliminal.png)`

Why is that? Do I need to insert some sort of escape character before quotation marks or slashes?

**Comment by thebaer** [comment]
Sorry for the delay on this! The forum will be the best place to discuss this, so I'll respond over there.
